nntrainer/graph/network_graph.h

```diff
@@ -292,14 +292,23 @@ class NetworkGraph {
   /**
    * @brief Allocate and initialize the weight variable
    */
-  void initializeWeights() { tensor_manager->initializeWeights(); }
+  void initializeWeights() {
+    tensor_manager->initializeWeights(
+      std::get<0>((*(cend() - 1))->getExecutionOrder()));
```

Comment:
Could you separate this to a variable or add a note? *(cend() - 1))->getExecutionOrder()

Reply:
will do.

Reply:
done.